Transformations/AESmethods.h

```diff
@@ -37,4 +37,18 @@ void invShiftRows(unsigned char state[4][4]);
 // Returns: Nothing
 // Notes: This is a private method
 void shiftRows(unsigned char state[4][4]);
+
+// Description: Substitutes the bytes of the state matrix
```

Comment:
This is really nice

Reply:
Mis click on my end. Also i cant get the Inverse Cypher to work.

Reply:
Lo unico que te puedo recomendar es ir full on debugging paso por paso con el ejemplo. Eso o me lo puedes pasar